scripts/README.md

```diff
@@ -0,0 +1,73 @@
+# Cardano SL Scripts
+
+This directory contains Bash scripts we use for different tasks (e.g. building, launching, CI).
+
+## Build
+
+* `build/cardano-sl.sh` - build Cardano SL, both in `dev` and `prod` modes.
+* `build/daedalus-bridge.sh` - build Daedalus Bridge, for work with Daedalus wallet.
+
+Please note that running mode depends on building mode! E.g. if you built Cardano SL in `dev`
+mode, it will run in `dev` mode as well, and if you built it in `prod` mode, it will run in
+`prod` mode as well.
+
+## Launch
+
+* `launch/demo.sh` - run nodes in `tmux`-session (3 nodes by default).
+* `launch/demo-with-wallet-api.sh` - run nodes in `tmux`-session, with enabled wallet web API (3 nodes by default).
+* `launch/kill-demo.sh` - kill `tmux`-session with running nodes.
+* `launch/qa.sh` - run one node, qanet.
+* `launch/update-scenario.sh` - scenario for testing of update mechanism.
+* `launch/wallet.sh` - helper script for `launch/update-scenario.sh`.
+
+## Bench
+
+* `bench/run-smart-generator.sh` - run [`cardano-smart-generator`](https://cardanodocs.com/technical/cli-options/#cardano-smart-generator).
+
+## Analyze
+
+* `analyze/blocks.sh` - analyze node logs: search information about block creation.
+* `analyze/block-events.sh` - analyze node logs: search information about different block-related events.
+
+## AVVM
+
+* `avvm-files/full_blacklist.js` - file for `cardano-keygen`. It contains list of blacklisted addresses.
+* `avvm-files/utxo-dump-last-new.json` - file for `cardano-keygen`. It contains AVVM stakes data.
+
+## Clean
+
+* `clean/db.sh` - clean Cardano SL DB data.
+* `clean/daedalus-bridge.sh` - clean Daedalus Bridge building artifacts.
+* `clean/all.sh` - do previous steps and clean `.stack-work` directory as well (in this case full rebuilding is required).
+
+## Generate
+
+* `generate/certificates.sh` - generate certificates using [`postvend-app`](https://github.com/input-output-hk/postvend-app). Please make sure you have `postvend-cli` command in your `PATH`.
+* `generate/genesis.sh` - generate keys using `cardano-keygen`.
+
+## Haskell
+
+* `haskell/lint.sh` - `hlint` command for Cardano SL source code. It uses `HLint.hs`-settings (from the project's root).
+* `haskell/stylish.sh` - `stylish-haskell` command for Cardano SL source code. It uses `HLint.hs` as well.
```

Comment:
No, it doesn't use it, it explicitly ignores it because we don't want to format `HLint.hs`.